models/endPoints.model.js

```diff
@@ -0,0 +1,12 @@
+const fs = require("fs/promises")
+
+exports.fetchEndPoints = () => {
```

Comment:
It's fine to achieve this with fs.readfile - however you could just require in the whole json object like you would any other file.